src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/dotnetcli.host.json

```diff
@@ -37,9 +45,25 @@
       "longName": "windowsAppSdk",
       "shortName": "win"
     },
-    "tests": {
-      "longName": "tests",
-      "shortName": "tests"
+    "unittest": {
+      "longName": "unittest",
+      "shortName": "unittest"
+    },
+    "uitest": {
+      "longName": "uitest",
+      "shortName": "uitest"
+    },
+    "configuration": {
+      "isHidden": true
```

Comment:
Since these (configuration, localization, logging) are just code blocks that we enable/disable, can we add these into this PR?